internal/pkg/cli/storage_init.go

```diff
@@ -212,6 +212,11 @@ func (o *initStorageOpts) Validate() error {
 		}
 	}
 
+	// auto-set noLSI if noSort specified.
+	if o.noSort {
```

Comment:
Hm, I've considered the `validate` method to be more of a read only method.

Reply:
Fixed in latest.